stack.yaml

```diff
@@ -3,3 +3,7 @@ resolver: lts-11.10
 flags:
   yaml:
     no-examples: false
+
+packages:
+  - .
```

Comment:
Let's put the `yaml` package in the subdir `yaml` as well if you don't mind.

Reply:
That's fine by me. Will do.